test/configs/oscap-customizations.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "oscap-customizations",
```

Comment:
RHEL7 and some of the earlier RHEL8 versions don't support openscap customizations... any ideas on how I could run these tests conditionally?

Reply:
Clearly, I didn't think this through well enough.  I'll have to include a `distro` field in the config changes I'm working on.

Reply:
Okay cool